src/utils/symlinkPackageDependencies.js

```diff
@@ -191,7 +185,10 @@ export default async function symlinkPackageDependencies(
       let src = path.join(depWorkspace.pkg.dir, String(binPath));
       let dest = path.join(pkg.nodeModulesBin, binName);
 
-      symlinksToCreate.push({ src, dest, type: 'exec' });
+      // Just in case the symlink is already added (it might have already existed in the projects bin/)
+      if (!symlinksToCreate.find(symlink => symlink.dest === dest)) {
```

Comment:
Just to explain this check:

I noticed that when running `install` twice in a row I'd get an error about a symlink already being created. Originally I assumed this must have been from my added code (which hypothetically should be fine, the symlink command rimrafs existing symlinks if they are there).

Upon investigation I found it it was happening in `symlinkPackageDependencies` because a symlink was being added to `symlinksToCreate` twice, once because it was in the root dependencies and once because it was an internal dependency.

This check was the least intrusive way to fix the issue.

A couple of other solutions could be: 
* Remove all the code that symlinks binaries between packages and let this be taken care of by `symlinkPackagesBinariesToProject`. It would copy all binaries from packages into the root .bin and then we'd automatically pick them all up anyway. There's no problem with this imo, the main difference would be that all packages would now have bin's from other internal packages, where as previously they would only get them if they had a dependency (this is not much different to all packages getting a link to all root external bins imo)
* Run the symlinking code in serial rather than parallel. Not a great solution, it would solve the problem because each time we tried to make a symlink it would always delete first if there was an existing one, this would be fairly slow however.
* We could rethink how we do all the symlinking of binaries into packages. This currently actually takes a very long time for some reason. We could investigate possibly symlinking the bin directory instead? This would be the most dangerous change imo.